### ISSUE TITLE
Lottie Animation works great in Android but not in IOS

### ISSUE THREAD

**Issue by vinuremesan17** [airbnb/lottie-ios#268]
in IOS the colors are inverted (but still animates) need a fix for this ...do let me know ASAP

**Comment by therealsalih** [comment]
@vinuremesan17 Can you send the AEP file?

**Comment by lookingstars** [comment]
I have the same problem. The animation json file works great in Android and  website https://www.lottiefiles.com/  ,but not in iOS
This is my json file:  
[overlay.json.zip](https://github.com/airbnb/lottie-ios/files/1096990/overlay.json.zip)

**Comment by buba447** [comment]
@lookingstars can you send an aep file?

**Comment by vinuremesan17** [comment]
Yeah Sure...
Just sending you a test showing whats happening in my animation.

if you see the AE file you will see two rings animating but in the json its
like two solid circles.
I used one more shape in front of each layer and submitted my animation but
thats an easy escape and alot of extra work.

can you let me know whats the probblem with this.

**Comment by blazejchwiecko** [comment]
Hey, I'm having exactly same issue. The animation looks stunning on demo file, in AfterEffects and even Android, but it doesn't seem to work on iOS, where instead of playing the whole screen it's getting white and animation is not visible. I've tried many different approaches, body movin' settings and struggled with it for a while, but can't make it work like it suppose to! He's my .aep file, .json and demo HTML. Please, tell me what's wrong with it and how to fix it!

[Archive.zip](https://github.com/airbnb/lottie-ios/files/1131287/Archive.zip)

**Comment by xtreme-david-joo** [comment]
@vinuremesan17 @buba447 Hello, I'm having the same issue.  If you could share what the solution was for this, that would be much appreciated.  Thanks!

**Comment by lookingstars** [comment]
@buba447  Can you help me
[2.aep.zip](https://github.com/airbnb/lottie-ios/files/1181976/2.aep.zip)
I download json file and use this way to load it, but it does't work in iOS
```
LOTAnimationView(contentsOf: URL(string: mainPath))
//mainPath:  file:///user/*******/**.json
```
THX

**Comment by jack-wz** [comment]
你好，我来自中国，不会用英语交流。
看了下你的文件，可能导致情况
文件中有位图，需要重新链接位图文件。


________________________________
发件人: ljz <notifications@github.com>
发送时间: 2017年7月28日 4:04
收件人: airbnb/lottie-ios
抄送: Subscribed
主题: Re: [airbnb/lottie-ios] Lottie Animation works great in Android but not in IOS (#268)


@buba447<https://github.com/buba447> Can you help me
2.aep.zip<https://github.com/airbnb/lottie-ios/files/1181976/2.aep.zip>
I download json file and use this way to load it, but it does't work in iOS

LOTAnimationView(contentsOf: URL(string: mainPath))
//mainPath:  file:///user/*******/**.json


THX

―

**Comment by JohnWong** [comment]
@lookingstars I think your problem is not about this issue. The path seems to be a path in you mac. It may be accessed in simulator but not in iPhone. Please try `[NSData dataWithContentsOfURL:url]` and see if data is loaded.

**Comment by JohnWong** [comment]
@lookingstars  I translate the reply from @jack-wz for you.

Hi, I'm from China, and can not communicate in English.
I have checked your file. Your file contains bitmap and this may be the reason. You may need to link the bitmap files again.

**Comment by lookingstars** [comment]
@JohnWong Should I have to load the local json file by this way?
```
LOTAnimationView(filePath: mainPath)
```
Lottie-ios: tag 1.5.1 seems doesn't support bitmap  which is link by json file ?

**Comment by JohnWong** [comment]
@lookingstars If you want to use NSURL and you just put the json file into main bundle, the code may be `[[[NSBundle mainBundle] bundleURL] URLByAppendingPathComponent:@"xxx.json"]`

**Comment by lookingstars** [comment]
@JohnWong 
Got it.  
If I want to user local json file I'd better user` lottie-ios tag 1.5.2`, and use it like this
`let animationV = LOTAnimationView(filePath: mainPath)`.
THX!

**Comment by buba447** [comment]
Should be fixed in Lottie 2.0! If you still have a problem please create a new issue!
